helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -365,6 +365,9 @@ logsCollection:
     #       value: server
     #     firstEntryRegex: ^[^\s].*
 
+    # Set `splunk.com/include` annotation to true on pod and `logsCollection.useSplunkIncludeAnnotation` flag to `true` to include its logs from ingested to your Splunk platform deployment.
+    # All other logs will be ignored. You cant use this feature with the above mentioned exclude feature. You can only use either the include feature or the exclude feature.
```

Comment:
Oh missed one last thing. There is nothing "above mentioned". Please rephrase :)